.circleci/config.yml

```diff
@@ -94,7 +94,7 @@ commands:
     description: 'Notify slack '
     steps:
       - slack/status:
-          failure_message: 'A $CIRCLE_JOB job has failed on master!'
+          failure_message: 'A $CIRCLE_JOB job has failed on $CIRCLE_BRANCH!'
           fail_only: true
           mentions: 'SRHGGRGS0' # Group ID for ActiveDevs
           only_for_branches: 'master'
```

Comment:
Not sure that this will work since the step has a branch filter for `master`

Reply:
@kerzhner  Good spot: any objection to removing that filter? Or might be nicer to just add 'deploy' to the filter?

Reply:
In general, the idea with that filter is that we do not want to send slack notifications for PR circle runs. We can either add `deploy` to the filter here or move the filtering outside of the `notify_slack` command definition.

Reply:
b7132fd